module/module.go

```diff
@@ -305,12 +305,17 @@ func (moduleSystem *System) GenerateBuild(
 				targetGenDir,
 				classInstance.Directory,
 			)
+			prettyBuildPath := filepath.Join(
```

Comment:
path.Rel will give example/example-gateway/build which is too much.

Just doing build/... is good enough.